fdbclient/ReadYourWrites.h

```diff
@@ -162,9 +162,9 @@ class ReadYourWritesTransaction : NonCopyable, public ReferenceCounted<ReadYourW
 	double creationTime;
 	bool commitStarted;
 
-	// If true, then this transactions write conflict range is not known until commit time.
-	// Currently only set if this transaction has a SetVersionstampedKey mutation
-	bool writeConflictRangeUnknown = false;
+	// For reading conflict ranges from the special key space
+	VectorRef<KeyRef> versionStampKeys;
+	Future<Standalone<StringRef>> versionStampFuture;
```

Comment:
Also keep a reference to the tr's conflict ranges so that we can still read conflict ranges after commit when ryw is disabled.